modules/generator/processor/spanmetrics/config.go

```diff
@@ -12,7 +12,16 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
-var targetInfoIntrinsicLabels = append(validation.SupportedIntrinsicDimensions, processor.DimJob, processor.DimInstance)
+var targetInfoIntrinsicLabelsSet map[string]struct{}
```

Comment:
I am bit confused by this change? and don't understand why?

I liked the []string as it was simple and reads better and doesn't need the setup we do in the `init()`